dvp: added CTA to the intro paragraph of insights and analytics (#16549)
docker/docs: docker-hub/publish/insights-analytics.md

Publisher (DVP) images on Docker Hub. With this tool, you have self-serve access
to metrics as both raw data and summary data for a desired time span. You can
view number of image pulls by tag or by digest, and get breakdowns by
geolocation, cloud provider, client, and more. Head to the
[Docker Verified Publisher Program page](https://www.docker.com/partners/programs/){: target="blank" rel="noopener" class="_" }

> to learn more about the benefits you get by becoming a verified publisher.

Reviewer: ```suggestion
to learn more about the benefits of becoming a verified publisher.
```
